nephos/fabric/crypto.py

```diff
@@ -326,6 +332,70 @@ def cacerts_to_secrets(namespace, msp_path, user):
         item_to_secret(namespace, msp_path, user, item)
 
 
+def tls_to_secrets(namespace, tls_path, username):
+    """Get the tls server.crt and server.key and create the relevant secret
+
+        Args:
+            namespace (str): Namespace where secret will live.
+            tls_path (str): Path to the tls crypto-material.
+            username (str): Username for identity.
+    """
+    keys_files_path = {"tls.crt": f"{tls_path}/server.crt", "tls.key": f"{tls_path}/server.key"}
+    secret_name = f"hlf--{username}-tls"
+    secret_from_files(secret=secret_name, namespace=namespace, keys_files_path=keys_files_path)
+
+    keys_files_path = {"cacert.pem": f"{tls_path}/ca.crt"}
+    secret_name = f"hlf--orderer-tlsrootcert"
+    secret_from_files(secret=secret_name, namespace=namespace, keys_files_path=keys_files_path)
+
+
+def setup_tls(opts, msp_name, release, id_type):
+    """Setup tls by saving ID to K8S secrets.
```

Comment:
And here "tls" -> "TLS". Naturally not python function names, but comments and DocStrings should be tidy. Thanks, @inzamam-iqbal